lighthouse-core/lib/log.js

```diff
@@ -113,23 +115,23 @@ class Log {
     Log._logToStdErr(`${prefix}:${level || ''}`, [data.method, snippet]);
   }
 
-  static log(title) {
-    Log.events.issueStatus(title, arguments);
-    return Log._logToStdErr(title, Array.from(arguments).slice(1));
+  static log(title, ...args) {
+    Log.events.issueStatus(title, ...args);
+    return Log._logToStdErr(title, args);
   }
 
-  static warn(title) {
-    Log.events.issueWarning(arguments);
-    return Log._logToStdErr(`${title}:warn`, Array.from(arguments).slice(1));
+  static warn(title, ...args) {
+    Log.events.issueWarning(title, ...args);
+    return Log._logToStdErr(`${title}:warn`, args);
   }
 
-  static error(title) {
-    return Log._logToStdErr(`${title}:error`, Array.from(arguments).slice(1));
+  static error(title, ...args) {
+    return Log._logToStdErr(`${title}:error`, args);
   }
 
-  static verbose(title) {
+  static verbose(title, ...args) {
     Log.events.issueStatus(title);
```

Comment:
looks like it was a bug introduced in https://github.com/GoogleChrome/lighthouse/commit/71195e4ac66f096c8505ca78321b523d9722270c#diff-e5b0c339f7ea6da4ec03ff079a28288cL100 I'll fix.